packages/components/src/icons/Phone.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { SVGProps } from "react";
+
+const SvgPhone = (props: SVGProps<SVGSVGElement>) => (
+  <svg width={17} height={16} {...props}>
```

Comment:
Default width and height is 24 x 24. If designers have provided something else, you need to get it fixed or fix yourself.

Reply:
changes are done. please check updated PR.

Reply:
The icons are not rendered correctly. You may want to run the generate-icons script.